modules/web-discovery-project/sources/web-discovery-project.es

```diff
@@ -1860,6 +1864,8 @@ const WebDiscoveryProject = {
           // and the canonical is not the same url, which comes out of automatic generation
           // of canonicals
           strict_value = false;
+        } else if (!structure && page_doc["alw"]) {
+          strict_value = false;
```

Comment:
I'd suggest adding a comment here to say that the value is allowlisted since the identifier `alw` may not be obvious to the reader.